ci/common.sh

```diff
@@ -56,6 +56,19 @@ rerun_flaky_tests() {
     FLUSH_ETCD=1 prove --timer -I./test-nginx/lib -I./ $(echo "$tests" | xargs)
 }
 
+install_curl () {
+    wget https://curl.se/download/curl-7.88.0.tar.gz
```

Comment:
What about exacting the version out like the other install_xxx function?

Reply:
Yes, it makes sense, fixed.